backend/trojan_dining/views.py

```diff
@@ -32,3 +36,23 @@ def get(self, request):
         sleep(5)
         user = getpass.getuser()
         return JsonResponse({"username": user})
+
+class GetMenu(APIView):
+    def get(self, request):
+        #get the date and if it doesnt exist return ''
+        date = request.GET.get('date', '')
+        #turn it into datetime
+        date = datetime.datetime.strptime(date, "%Y-%m-%d")
```

Comment:
What happens if the date is malformatted? Can we make sure this doesn't crash the server and return a 400 status code? (`return HttpResponse(status=204)`)

Reply:
done